src/common.rs

```diff
@@ -111,6 +111,27 @@ pub enum BinaryFormat {
     Pe,
     Wasm,
     Xcoff,
+    Host,
+}
+
+impl BinaryFormat {
+    /// Returns the BinaryFormat of the enum
+    /// Used for unwraping the `Host` option
+    #[allow(dead_code)]
+    pub fn get(self) -> BinaryFormat {
```

Comment:
```suggestion
    /// The target's native binary format for relocatable object files.
    ///
    /// Defaults to `Elf` for unknown platforms.
    pub fn native_object() -> BinaryFormat {
```

Instead of adding the `Host` variant, change this to be a method that directly returns the format, so then the user can use `BinaryFormat::target_object()` instead of `BinaryFormat::Host`.